server/sonar-server/src/test/java/org/sonar/server/qualityprofile/ws/QProfilesWsTest.java

```diff
@@ -113,7 +107,7 @@ public void define_controller() {
     assertThat(controller).isNotNull();
     assertThat(controller.path()).isEqualTo(QProfilesWs.API_ENDPOINT);
     assertThat(controller.description()).isNotEmpty();
-    assertThat(controller.actions()).hasSize(23);
+    assertThat(controller.actions()).isNotEmpty();
```

Comment:
QProfilesWsTest should be dropped. All the definition tests should be moved to the action tests.